web/js/map/animation/ui.js

```diff
@@ -313,12 +356,13 @@ export function animationUi(models, ui) {
    * @returns {boolean}
    *
    */
-  self.isInToPlayGroup = function (testDate) {
+  self.isInToPlayGroup = function(testDate) {
     var loop = animModel.rangeState.loop;
     var i = 0;
     var day = new Date(self.state.playIndex);
     var startDate = new Date(animModel.rangeState.startDate);
     var endDate = new Date(animModel.rangeState.endDate);
+    // console.log(startDate, endDate);
```

Comment:
```suggestion

```

Reply:
Hmm.. This file should be gone. I think it was added back in with a merge/rebase.

Reply:
✅